test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -184,33 +213,43 @@ describe.skip('Create token, approve token and approve token without gas', funct
           dapp: true,
           fixtures: 'connected-state',
           ganacheOptions,
+          smartContract,
+          failOnConsoleError: false,
           title: this.test.title,
         },
-        async ({ driver }) => {
+        async ({ driver, contractRegistry }) => {
+          const contractAddress = await contractRegistry.getContractAddress(
+            smartContract,
+          );
           await driver.navigate();
           await driver.fill('#password', 'correct horse battery staple');
           await driver.press('#password', driver.Key.ENTER);
 
-          await driver.openNewPage(`http://127.0.0.1:8080/`);
+          // create token
+          await driver.openNewPage(
+            `http://127.0.0.1:8080/?contract=${contractAddress}`,
+          );
 
-          await driver.waitForSelector({ text: 'Create Token', tag: 'button' });
-          await driver.clickElement({ text: 'Create Token', tag: 'button' });
-          windowHandles = await getWindowHandles(driver, 3);
-          await driver.switchToWindow(windowHandles.popup);
-          await driver.clickElement({ text: 'Confirm', tag: 'button' });
+          let windowHandles = await driver.getAllWindowHandles();
+          const extension = windowHandles[0];
 
-          await driver.switchToWindow(windowHandles.dapp);
+          await driver.findClickableElement('#deployButton');
 
+          // approve token from dapp
           await driver.waitForSelector({
```

Comment:
we can remove this await driver.waitForSelector('...'); for same reason above

Reply:
done